server/lib/nl/page_config_next.py

```diff
@@ -21,17 +21,44 @@
 from config.subject_page_pb2 import SubjectPageConfig
 from config.subject_page_pb2 import Tile
 from lib.nl import utils
+from lib.nl.detection import EventType
 from lib.nl.detection import Place
 from lib.nl.detection import RankingType
 from lib.nl.utterance import ChartSpec
 from lib.nl.utterance import ChartType
 from lib.nl.utterance import Utterance
 
+# NOTE: This relies on disaster config's event_type_spec IDs.
```

Comment:
Maybe both the textproto and here should use enum defined in proto

Reply:
Which enum in proto, could you point to it?

Reply:
I am suggesting to create an Enum in the proto, as now they are free formed key?

Reply:
Oh, I thought you were suggesting a pre-existing enum.  That is a good idea, I'll add a TODO.

@chejennifer or @beets for inputs/scheduling

Reply:
I don't think having an enum in the proto for event type ids makes the most sense because the event_type_spec was designed to be created by whoever is creating that page's config. I would think of it like the stat_var_spec where the caller adds an event_type_spec with the id "x" & then includes "x" in the list of event_type_keys for tiles where that event type should be shown. 

I would actually suggest not relying on disaster dashboard's config, but instead, maybe have a constant dict somewhere where key is EventType and value is all the field values for the event_type_spec we want for that EventType. Then when adding an event chart block, use this constant to add required event_type_specs to the page metadata.

Reply:
I actually started with the latter approach, and found myself duplicating proto config in py code, and it looked ugly :)

I slightly like the plan that @beets suggests better, of using textproto, but forking it away from disasters.

Reply:
ah, I see, so a .textproto file with all the event_type_specs specific for NL? that makes sense to me!

Reply:
yup!